string arm
float32 position
float32 velocity
string units